### DISCUSSION TITLE
Avoid sorting folders before files

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [jarun/nnn#1838] [General]
Is there a  way to avoid sorting folders before files, and sort everything together?

**Comment by jarun** [comment]
No. There are no plans to do so either.

**Comment by raffaelemancuso** [comment]
> No. There are no plans to do so either.

But why close the issue? Maybe someone would like  to submit a PR

**Reply by jarun** [reply]
As I mentioned, there are no plans to do this. There were requests for this feature earlier as well. But I want to stick to the current interface.
